pkg/resources/stage_grant_test.go

```diff
@@ -0,0 +1,61 @@
+package resources_test
+
+import (
+	"database/sql"
+	"testing"
+	"time"
+
+	sqlmock "github.com/DATA-DOG/go-sqlmock"
+	"github.com/chanzuckerberg/terraform-provider-snowflake/pkg/provider"
+	"github.com/chanzuckerberg/terraform-provider-snowflake/pkg/resources"
+	. "github.com/chanzuckerberg/terraform-provider-snowflake/pkg/testhelpers"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStageGrant(t *testing.T) {
+	r := require.New(t)
+	err := resources.StageGrant().InternalValidate(provider.Provider().Schema, true)
+	r.NoError(err)
+}
+
+func TestStageGrantCreate(t *testing.T) {
+	a := assert.New(t)
+
+	in := map[string]interface{}{
+		"stage_name":    "test-stage",
+		"schema_name":   "PUBLIC",
+		"database_name": "test-db",
+		"privilege":     "USAGE",
```

Comment:
Same as above - to be sure that we correctly handle non-default parameters, we should test with a non-default.

Reply:
Added a few stage access control privileges that I missed (https://docs.snowflake.net/manuals/user-guide/security-access-control-privileges.html#stage-privileges) and added a non-default test.